Update microsoft-tunnel-configure.md (#3011)
MicrosoftDocs/memdocs: memdocs/intune/protect/microsoft-tunnel-configure.md

## Uninstall the Microsoft Tunnel

To uninstall the product, run **./mst-cli uninstall** from the Linux server as root. 

> Once succesfully uninstalled, delete the corresponding server record in the [Microsoft Endpoint Manager admin center](https://go.microsoft.com/fwlink/?linkid=2109431) under **Tenant administration** > **Microsoft Tunnel Gateway** > **Servers**.

Reviewer: ```suggestion
After the product is uninstalled, delete the corresponding server record in the [Microsoft Endpoint Manager admin center](https://go.microsoft.com/fwlink/?linkid=2109431) under **Tenant administration** > **Microsoft Tunnel Gateway** > **Servers**.
```
